platform/ios/src/MGLMapView.mm

```diff
@@ -4935,8 +4935,10 @@ - (void)updateAnnotationViews
 
     // If the map is pitched consider the viewport to be exactly the same as the bounds.
     // Otherwise, add a small buffer.
-    CGFloat widthAdjustment = self.camera.pitch > 0.0 ? 0.0 : -_largestAnnotationViewSize.width * 2.0;
-    CGFloat heightAdjustment = self.camera.pitch > 0.0 ? 0.0 : -_largestAnnotationViewSize.height * 2.0;
+    CGFloat largestWidth = _largestAnnotationViewSize.width ?: CGRectGetWidth(self.frame);
```

Comment:
I don’t even want to think about what would happen if `_largestAnnotationViewSize` wound up negative. But I think it would better express your intention here to use `MAX()` instead of `?:`.